pandas/tests/reshape/test_pivot.py

```diff
@@ -781,6 +781,15 @@ def test_pivot_with_list_like_values_nans(self, values, method):
         expected = DataFrame(data=data, index=index, columns=columns, dtype="object")
         tm.assert_frame_equal(result, expected)
 
+    def test_pivot_columns_none_raise_error(self):
+        # GH 30924
+        df = pd.DataFrame(
+            {"col1": ["a", "b", "c"], "col2": [1, 2, 3], "col3": [1, 2, 3]}
+        )
+        msg = 'pivot["(][")] missing 1 required argument: ["\']columns["\']'
```

Comment:
if you use re.escape, this may be more readable.

Reply:
thanks for your quick response, i tried `re.escape`, but it seemed to work in an opposite way. @simonjayhawkins could you pls enlighten me a bit?